lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
```

Comment:
> exposing this to the rule somehow

Something like this?

```js
module.exports.meta = {
    errors: {
        GETTER: "Getter is not present",
        SETTER: "Setter is not present"
    }
    // ...
};
```

From there, two options:

```js
// First option: core passes `errors` to the rule, which references the messages by key
context.report(node, context.errors.GETTER);
// Second opiton: rule references the error message in `context.report` by key
context.report(node, "GETTER");
```

The first looks cleaner to me, but the second would do a better job of enforcing that rules don't supply their own error messages out of sync with the metadata.